### PR TITLE
Jinja2 mode Line statements support

### PR DESCRIPTION
**Pull request by vled12**
According to Jinja2 [template designer manual](https://jinja.palletsprojects.com/en/3.1.x/templates/#line-statements), the line statements are now supported. Test example is included.
Beside that, keyword list is slightly updated.

### CODE REVIEW

**Comment by Jay4577** [comment]
Pullrequst

**Review by Jay4577 — APPROVED** [review]

File: mode/jinja2/jinja2.js [hunk 1]
```diff
@@ -78,7 +78,24 @@
           state.instring = ch;
           stream.next();
           return "string";
-        } else if(stream.match(state.intag + "}") || stream.eat("-") && stream.match(state.intag + "}")) {
+        }
+        else if (state.inbraces > 0 && ch ==")") {
```
Comment (marijnh) [inline mode/jinja2/jinja2.js:82]: Can you make sure `inbraces`/`inbrackets` are initialized in the start state?

Comment (vled12) [reply mode/jinja2/jinja2.js:82]: `inbraces / inbrackets` are accessed only in `intag` state.
Currently they are initialized right after the turning `intag` state on. I assume, this should be sufficient.

Comment (marijnh) [reply mode/jinja2/jinja2.js:82]: It seems dodgy to compare `undefined > 0` here (though it will in fact produce false)

**Comment by marijnh** [comment]
Sure that works, but why not just initialize them to 0 in `startState` and avoid the question of whether they are defined entirely?

**Comment by marijnh** [comment]
Thanks. Merged.

**Pull request merged**